docs/blog/newsletter-july-2021.rst

```diff
@@ -0,0 +1,122 @@
+
+.. post:: July 06, 2021
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2021
+#####################################
+
+Good morning, afternoon, or evening, documentarians! It's Beth and the newsletter team with a last summer update before we head off on a little aestival break. We'll be back in September :) 
+
+The main announcement for the month is that the `CFP and ticket sales </conf/australia/2021/news/cfp-open/>`__ have opened for the Australia and India conference! As with our other conferences this year, it'll be held online, and the dates for that are 2nd-3rd December. The CFP is open until 15th August, so get thinking about talks you'd like to submit!
+
+Onto our stories from Slack...
+
+-----------------------------------------
+What's best for your images - JPG or PNG?
+-----------------------------------------
+
+JPG or PNG? Which is the best format to use? Does it even matter which you pick? Well, the Write the Docs community has chimed in to answer your pressing questions, and it seems like PNG is the clear winner. `Portable Network Graphics <https://en.wikipedia.org/wiki/Portable_Network_Graphics>`__, or PNG, offers unmatched clarity for screenshots in online docs and doesn’t degrade when opened and saved like the alternative JPG or JPEG. Compression in PNG is also better since it’s a lossless image format and not a lossy one like JPG, meaning it preserves all of an image’s details. Another advantage of PNG is that it supports transparency, whereas JPG is best for gradient images.
```

Comment:
Feels odd to define and link PNG but not JPG?

Reply:
added